api/editor/.editorconfig

```diff
@@ -0,0 +1,26 @@
+# EditorConfig is awesome: https://EditorConfig.org
```

Comment:
Another attempt to fix linting issues with java applications, but it doesn't seem to work.